### DISCUSSION TITLE
Control Music Assistant using NSPanel Lovelance UI

### DISCUSSION THREAD

**Discussion by ZVlog** [music-assistant/support#1610] [Feature requests and ideas]
Does anyone have any ideas how to do this?

Selecting playlists, songs, radio stations

**Comment by OzGav** [comment]
I’m not exactly sure what you are asking? You can use scripts and automations to  control MA. They could be triggered by buttons?

**Comment by AviFix** [comment]
> Does anyone have any ideas how to do this?
> 
> Selecting playlists, songs, radio stations

Did you get it working?
